frontend/app/module/data_entry/page/DataEntryHomePage.tsx

```diff
@@ -17,7 +17,9 @@ export function DataEntryHomePage() {
 
   // re-fetch statuses when component mounts
   useEffect(() => {
-    refetch();
+    refetch().catch((e: unknown) => {
+      throw e;
+    });
```

Comment:
This is nicer and seems to work for me:
```suggestion
    void refetch();
```

Reply:
Haha - thank you! Just changed the code in the same way and then found your excellent suggestion :+1: